core/src/main/java/com/opensymphony/xwork2/config/providers/XWorkConfigurationProvider.java

```diff
@@ -224,6 +224,7 @@ public void register(ContainerBuilder builder, LocatableProperties props)
         props.setProperty(StrutsConstants.STRUTS_ENABLE_OGNL_EXPRESSION_CACHE, Boolean.TRUE.toString());
         props.setProperty(StrutsConstants.STRUTS_ENABLE_OGNL_EVAL_EXPRESSION, Boolean.FALSE.toString());
         props.setProperty(StrutsConstants.STRUTS_CONFIGURATION_XML_RELOAD, Boolean.FALSE.toString());
+        props.setProperty(StrutsConstants.STRUTS_ALLOW_STATIC_METHOD_ACCESS, Boolean.FALSE.toString());
```

Comment:
Why? Is it not working?

Reply:
Hello @aleksandr-m .
Are you asking why the new entry on line 227 ?  Simply to ensure the props includes a set property for that element, in line with the other significant ones already present.

Reply:
But there is more constants. Why to add especially this one?

Reply:
Properties like devMode, DMI, eval_expression are all "privilege-related" (and set to false), just like static method access.  Since the others were present, it seemed that static method access should be there too (for completeness, also set to false).
That was my reasoning anyway.  :)